uktrade/data-hub-frontend
# Dashboard filter by status and land date

## Description of change

Adds filter by status and land date to the personalised dashboard

## Test instructions

Enable the personalised dashboard feature flag and try the filters above the project list. The summary chart etc should also be updated accordingly.

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/1234577/135070812-f4c85bf2-c9a9-4c7b-bad2-f27a788c9c0d.png)

### After

![image](https://user-images.githubusercontent.com/1234577/135070753-edcfeae7-666b-4181-af9d-525d5bcd1534.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
